ZAPD/Globals.cpp

```diff
@@ -177,11 +178,44 @@ void Globals::AddSegment(int32_t segment, ZFile* file)
 	if (std::find(segments.begin(), segments.end(), segment) == segments.end())
 		segments.push_back(segment);
 
-	segmentRefs[segment] = file->GetXmlFilePath();
+	segmentRefs[segment] = file->GetXmlFilePath().string();
 	segmentRefFiles[segment] = file;
 }
 
 bool Globals::HasSegment(int32_t segment)
 {
 	return std::find(segments.begin(), segments.end(), segment) != segments.end();
 }
+
+std::map<std::string, ExporterSet*>& Globals::GetExporterMap()
+{
+	static std::map<std::string, ExporterSet*> exporters;
+	return exporters;
+}
+
+void Globals::AddExporter(std::string exporterName, ExporterSet* exporterSet)
+{
+	std::map<std::string, ExporterSet*>& exporters = GetExporterMap();
```

Comment:
you can use `auto` here if you like.
```suggestion
	auto exporters = GetExporterMap();
```